Configurations/unix-Makefile.tmpl

```diff
@@ -674,6 +674,12 @@ update: generate errors ordinals
 generate: generate_apps generate_crypto_bn generate_crypto_objects \
           generate_crypto_conf generate_crypto_asn1
 
+doc-nits:
+	$(PERL) util/find-doc-nits.pl -n >doc-nits
+	if [ ! -s doc-nits ] ; then exit 0 ; fi
```

Comment:
it works, and we have other ```if [ ! ...``` tests in the makefile, but I made the change you said.